src/newton.js

```diff
@@ -0,0 +1,17 @@
+import {abs, epsilon} from "./math";
+
+// Newton-Raphson
+// Solve f(x) = 0, start from x = 0
+export function solve(f, y, start, steps) {
+  y = y || 0;
+  steps = steps || 100;
+  var x = start || 0, delta, f0, f1;
```

Comment:
This function is only called in one place at the moment, so it seems a bit premature to parameterize it with *start* and *steps*.

Also, since *y* comes from the user (via *projection*.invert), I recommend coercing it to a number (`y = +y`) and not silently replacing NaN with zero.

Reply:
The same function is used in _hufnagel_ and uses _start_
https://github.com/d3/d3-geo-projection/pull/147/files#diff-0d12b8fb830a005aa9dd7da6e2176c8cR40

(_steps_ are not used, so I will remove that parametrization).